src/GraphResponseHandler.ts

```diff
@@ -188,7 +221,10 @@ export class GraphResponseHandler {
 				}
 			}
 		} catch (error) {
-			throw rawResponse.ok ? error : rawResponse;
+			if (rawResponse.ok) {
+				throw error;
+			}
+			throw rawResponse;
```

Comment:
This change can be omitted - i had a consumeResponse in the ok branch in an earlier version of this change